src/main/java/essencepouchtracking/EssencePouchTrackingPlugin.java

```diff
@@ -128,6 +133,26 @@ public class EssencePouchTrackingPlugin extends Plugin
 	private int lastCraftRuneTick;
 	private int lastRCXP = -1;
 
+	private final Set<Integer> RC_CAPES_SET = ImmutableSet.<Integer>builder().addAll(this.getItemVariants(ItemID.RUNECRAFT_CAPE))
+		.addAll(this.getItemVariants(ItemID.MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.ACCUMULATOR_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.ASSEMBLER_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.MASORI_ASSEMBLER_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.DIZANAS_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.FIRE_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.INFERNAL_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.IMBUED_GUTHIX_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.IMBUED_SARADOMIN_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.IMBUED_ZAMORAK_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.MYTHICAL_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.GUTHIX_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.SARADOMIN_MAX_CAPE))
+		.addAll(this.getItemVariants(ItemID.ZAMORAK_MAX_CAPE))
+		.build();
```

Comment:
Could refactor and remove this

Although by doing this, users won't know exactly which capes are supported, or if Jagex changes requirements